csharp/src/Ice/Ice2Definitions.cs

```diff
@@ -29,7 +30,8 @@ internal enum FrameType : byte
             string facet,
             IReadOnlyList<string> location,
             string operation,
-            bool idempotent)
+            bool idempotent,
+            TimeSpan deadline)
```

Comment:
I would pass directly the Deadline date time here and not introduce a confusing intermediary TimeSpan deadline parameter.

Reply:
Fixed